spec/integration/aws_elasticsearch_domain_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+describe Chef::Resource::AwsElasticsearchDomain do
+  extend AWSSupport
+  when_the_chef_12_server "exists", organization: "foo", server_scope: :context do
+    with_aws "when connected to AWS" do
+
+      time = DateTime.now.strftime('%Q')
+
+      it "aws_elasticsearch_domain 'test-#{time}' creates a elasticsearch domain" do
```

Comment:
Thanks for writing a spec for this!  Can you add some more coverage to match our [resource acceptance criteria](https://github.com/chef/chef-provisioning-aws/blob/master/CONTRIBUTING.md#resourceprovider-acceptance-criteria)?  Basically, you need 1 test that sets all possible attributes, 1 test that checks that on an existing object all possible fields are updatable, and 1 test to validate the `:destroy` action.

An example of this is [load_balancer](https://github.com/chef/chef-provisioning-aws/blob/13adbadf798f562b8a9c76a0a60a5dd1383b47f6/spec/integration/load_balancer_spec.rb#L27-L269) specs (although yours should be simpler because your resource does not reference other resources).